canonical-websites/snapcraft.io
# Updates empty search results to show featured snaps

Fixes #768 

### QA

- ./run
- go to Store (or search) and search for something that doesn't exist
- search term should not be repeated in heading
- if there are no results featured snaps should be shown

<img width="1042" alt="screen shot 2018-06-28 at 13 19 19" src="https://user-images.githubusercontent.com/83575/42031360-48a8d30a-7ad6-11e8-93c4-fbd8bbf86a39.png">
